sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/models/ChatMessageDeletedEvent.java

```diff
@@ -12,28 +14,33 @@
      * The timestamp when the message was deleted. The timestamp is in ISO8601 format:
      * `yyyy-MM-ddTHH:mm:ssZ`.
      */
+    @JsonProperty(value = "deletedOn")
```

Comment:
have we considered using `OffsetDataTime` for `deletedOn` property in `ChatMessageDeletedEvent` type?

Reply:
[same comment for eligible date properties in other event models]